packages/framework-core/src/booster-token-verifier.ts

```diff
@@ -54,6 +62,13 @@ class TokenVerifierClient {
     return new Promise((resolve, reject) => {
       jwt.verify(token, key, this.options, (err, decoded) => {
         if (err) {
+          if (err instanceof TokenExpiredError) {
+            return reject(new BoosterTokenExpiredError(err.message))
+          }
+
+          if (err instanceof NotBeforeError) {
+            return reject(new BoosterTokenNotBeforeError(err.message))
+          }
```

Comment:
We created new exceptions to avoid returning jwt exceptions to the user. The idea is to start using it from the first jwt error to avoid processing jwt exceptions on the validator. Thoughts?

Reply:
Thinking about it, I will only use our own exceptions for returning the result to the user and remove them from here